torch/testing/_internal/common_utils.py

```diff
@@ -2439,6 +2440,265 @@ def genSparseTensor(self, size, sparse_dim, nnz, is_uncoalesced, device, dtype):
             x = x.detach().clone()._coalesced_(False)
         return x, x._indices().clone(), x._values().clone()
 
+    def generate_simple_inputs(self, layout,
+                               device=None,
+                               dtype=None,
+                               index_dtype=None,
+                               enable_batched=True,
+                               enable_hybrid=True,
+                               enable_zero_sized=True,
+                               enable_batched_variable_nse=False,
+                               output_tensor=True,
+                               patterns=None):
```

Comment:
What do you think about having `low/high` parameters for values?

Reply:
The aim is to have inputs where all values are distinct (to ease debugging and the development of sparse algorithms). I think `low/high` parameters make sense for randomly generated samples but here we want the inputs to be highly predictable.

Reply:
Knowing the range of values could be useful to control precision. The values generated seem to be only unique when each size `<10`, which is the case I assume, right?

Reply:
yes, the generated values aim at being unique. Notice the sizes of the generated tensors are intentionally small so that a human could easily track how the tensor elements transform when applying sparse tensor conversion/indexing and other tools that act mostly on the tensor structure rather than on the values.

For tests where precision parameters matter, one should use the `make_tensor` or other tensor generators that provide the low/high parameter feature.